hera_mc/geo_handling.py

```diff
@@ -194,12 +194,16 @@ def find_antenna_at_station(self, station, query_date):
         ctr = 0
         for conn in connected_antenna:
             if conn.stop_gpstime is None or query_date.gps <= conn.stop_gpstime:
+                if ctr:
+                    print("Already connected: {} -> {}".format(antenna_connected.upstream_part, antenna_connected.downstream_part))
+                    print("...trying to add:  {} -> {}".format(conn.upstream_part, conn.downstream_part))
                 antenna_connected = copy.copy(conn)
                 ctr += 1
         if ctr == 0:
             return None, None
         elif ctr > 1:
-            raise ValueError('More than one active connection between station and antenna')
+            warnings.warn('More than one active connection between station and antenna')
```

Comment:
can you explain the use case here? Seems pretty bad (and hard to interpret) to me if there are multiple connections. Also, your docstring says it raises a ValueError in this case, not a warning. The docstring should be updated.

Reply:
I fixed the docstring.  A warning and error convey the same information, and the warning is preferable since it can handle all non-warned ones (which may take some time and you wouldn't want to repeat).  Maintenance of this is an on-going thing.

Reply:
Ok. Is there an error somewhere that will force you (not the random user trying to make this plot) to address it? Errors are much more effective at causing change than warnings.

Reply:
I feel confident that a warning will suitably motivate me.

Reply:
ok